code/modules/economy/economy_machinery/eftpos.dm

```diff
@@ -218,3 +219,80 @@
 	R.forceMove(D)
 	D.wrapped = R
 	D.name = "small parcel - 'EFTPOS access code'"
+
+/obj/item/eftpos/proc/check_user_position(mob/user)
+	return Adjacent(user)
+
+/obj/item/eftpos/register
+	name = "point of sale"
+	desc = "Also known as a cash register, or, more commonly, \"robbery magnet\". It's old and rusty, and had an EFTPOS module fitted in it."
+	icon = 'icons/obj/machines/pos.dmi'
+	icon_state = "pos"
+	force = 10
+	throwforce = 10
+	throw_speed = 1.5
+	throw_range = 7
+	anchored = TRUE
+	w_class = WEIGHT_CLASS_BULKY
+	hitsound = 'sound/weapons/ringslam.ogg'
+	drop_sound = 'sound/items/handling/register_drop.ogg'
+	pickup_sound =  'sound/items/handling/toolbox_pickup.ogg'
+	transaction_sound = 'sound/machines/checkout.ogg'
+	attack_verb = list("bounced a check off", "checked-out", "tipped")
+
+/obj/item/eftpos/register/examine(mob/user)
+	. = ..()
+	if(!anchored)
+		. += "<span class='notice'>Alt-click to rotate it.</span>"
+	else
+		. += "<span class='notice'>It is secured in place.</span>"
+
+/obj/item/eftpos/register/AltClick(mob/user)
+	if(user.incapacitated())
+		to_chat(user, "<span class='warning'>You can't do that right now!</span>")
+		return
+	if(!Adjacent(user))
+		return
+	if(anchored)
+		to_chat(user, "<span class='warning'>[src] is secured in place!</span>")
+		return
+	setDir(turn(dir, 90))
+
+/obj/item/eftpos/register/attack_hand(mob/user)
+	if(anchored)
+		if(!check_user_position(user))
+			to_chat(user, "<span class='warning'>You need to be behind [src] to use it!</span>")
+			return
+		add_fingerprint(user)
+		ui_interact(user, state = GLOB.human_adjacent_state)
+		return TRUE
+	return ..()
+
+/obj/item/eftpos/register/attack_self(mob/user)
+	to_chat(user, "<span class='notice'>[src] has to be set down and secured to be used.</span>")
+
+/obj/item/eftpos/register/check_user_position(mob/user)
+	if(!..())
+		return FALSE
+	var/user_loc = get_dir(src, user)
+	if(!user_loc || user_loc & dir)
+		return TRUE
+	return FALSE
+
+/obj/item/eftpos/register/scan_card(obj/item/card/id/C, mob/user, secured = anchored)
+	..()
```

Comment:
this is pointless

Reply:
What do you mean? I'm passing `anchored` as a argument, but otherwise not changing anything else. I still have to call parent.

Reply:
Ah i missed that. Can you move the override into the ..(`overriden args`)